Multisig/UI/Transaction/ExecuteTransaction/PendingTransactionMontior.swift

```diff
@@ -128,10 +128,10 @@ class ChainPendingTransactionMonitor {
         // the core data objects might change while we're making the network request
         // therefore we need to remember the data that will tell us that we still should udpate the objects
         // after network request is completed.
-        let cdTxByTxHash: [String: (status: String, updatedAt: Date?, ethTx: CDEthTransaction)] = Dictionary(
-            uniqueKeysWithValues: cdEthTransactions.map { ($0.ethTxHash!, ($0.status!, $0.dateUpdatedAt, $0)) }
-        )
-
+        //
+        // patch: using array instead of a dictionary (by tx hash) because of duplicate entries in the database
```

Comment:
As we remove the duplication in the above file, do we still need this change here ? or just for safer change?

Reply:
out there in production, on user device, the database still has duplicate entries. This will lead to a crash if we won't handle it here.